web/templates/base.html

```diff
@@ -89,5 +89,6 @@
 		<script type="text/javascript" src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 		<script type="text/javascript" src="{% static 'js/countdown.js' %}"></script>
 		{% block custom_js %}{% endblock custom_js %}
+		<script type="text/javascript" src="{% static 'js/script.js' %}"></script>
```

Comment:
In this case custom_js have the script link to the datetimepicker.js. I thanked, that you can not reference something in script.js, that will be loaded after...